app/coffee/Features/Collaborators/CollaboratorsInviteController.coffee

```diff
@@ -0,0 +1,103 @@
+ProjectGetter = require "../Project/ProjectGetter"
+LimitationsManager = require "../Subscription/LimitationsManager"
+UserGetter = require "../User/UserGetter"
+Project = require("../../models/Project").Project
+User = require("../../models/User").User
+CollaboratorsInviteHandler = require('./CollaboratorsInviteHandler')
+mimelib = require("mimelib")
+logger = require('logger-sharelatex')
+
+module.exports = CollaboratorsInviteController =
+
+	getAllInvites: (req, res, next) ->
+		projectId = req.params.Project_id
+		logger.log {projectId}, "getting all active invites for project"
+		CollaboratorsInviteHandler.getAllInvites projectId, (err, invites) ->
+			if err?
+				logger.err {projectId}, "error getting invites for project"
+				return next(err)
+			res.json({invites: invites})
+
+	inviteToProject: (req, res, next) ->
+		projectId = req.params.Project_id
+		email = req.body.email
+		sendingUserId = req.session?.user?._id
+		logger.log {projectId, email, sendingUserId}, "inviting to project"
+		LimitationsManager.canAddXCollaborators projectId, 1, (error, allowed) =>
+			return next(error) if error?
+			if !allowed
+				logger.log {projectId, email, sendingUserId}, "not allowed to invite more users to project"
+				return res.json {invite: null}
+			{email, privileges} = req.body
+			email = mimelib.parseAddresses(email or "")[0]?.address?.toLowerCase()
```

Comment:
This looks familiar from elsewhere. Perhaps pull out into a little _parseEmail helper? We should standardise and use the same email parsing for login, register, invite, etc and this feels non-DRY atm.